internal/settings/config_provider.go

```diff
@@ -83,6 +92,18 @@ func (c *configProvider) GetExcludePrefixesForDir(dirPath string) ([]string, err
 	return getExcludePrefixesForDir(dirPath)
 }
 
+func (c *configProvider) GetExcludePrefixesForData(dirPath string, externalConfigData string) ([]string, error) {
+	if !filepath.IsAbs(dirPath) {
```

Comment:
Should we have this validation in `GetForData`, too?

Reply:
Sure